include/wx/msw/missing.h

```diff
@@ -150,6 +150,10 @@
 #define BS_TYPEMASK 0xf
 #endif
 
+#ifndef COM_DECLSPEC_NOTHROW
+    #define COM_DECLSPEC_NOTHROW
+#endif
+
```

Comment:
I hadn't known about this one before, thanks. After looking at it now, it seems that Microsoft simply forgot to update `STDMETHODIMP` and `STDMETHODIMP_()` definitions to include it when adding to `STDMETHOD` and `STDMETHOD_()`. So I think it might be better to just redefine those here to use `COM_DECLSPEC_NOTHROW` instead if it is defined, otherwise the definition of any COM method becomes even more verbose when the whole point of `STDMETHODIMP` macro is to make it shorter.

Alternatively, we could define `wxSTDMETHODIMP` and use it instead.